client/signup/steps/woocommerce-install-confirm-wordpress-subdomain/index.tsx

```diff
@@ -0,0 +1,141 @@
+import { NextButton } from '@automattic/onboarding';
+import { Spinner } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { ReactElement, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import yourNewStoreImage from 'calypso/assets/images/woocommerce-install/your-new-store.png';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import {
+	fetchAutomatedTransferStatusOnce,
+	requestEligibility,
+} from 'calypso/state/automated-transfer/actions';
+import {
+	isFetchingAutomatedTransferStatus,
+	getEligibility,
+	EligibilityData,
+} from 'calypso/state/automated-transfer/selectors';
+import { getSiteDomain } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import type { GoToStep } from '../../types';
+import type { AppState } from 'calypso/types';
+
+import './style.scss';
+interface Props {
+	goToStep: GoToStep;
+	isReskinned: boolean;
+}
+
+export default function ConfirmWordPressSubdoamin( {
+	goToStep,
+	isReskinned,
+}: Props ): ReactElement | null {
+	const siteId = useSelector( getSelectedSiteId ) as number;
+
+	const fetchingTransferStatus = !! useSelector( ( state: AppState ) =>
```

Comment:
TIL `useSelector` is a replacement for `mapStateToProps`. It also [takes a `shallowEqual` argument](https://react-redux.js.org/api/hooks#equality-comparisons-and-updates) to re-render even more efficiently, in case that's useful here.

Reply:
Thanks, Allison. Going to take a look and add it in case it improves the rendering performance. 🙇‍♂️

Reply:
I've added the shallowEqual function here https://github.com/Automattic/wp-calypso/pull/58024/commits/ea39a65ae5abaa4504f2d7abfdc4b798a91f0fd3